apps/src/templates/instructions/utils.js

```diff
@@ -114,8 +114,15 @@ function removeCommentNodes(root) {
  * @param {Boolean} isRtl True if we are displaying in RTL
  */
 export function convertXmlToBlockly(xmlContainer, isRtl) {
-  const xmls = xmlContainer.getElementsByTagName('xml');
+  // Remove any divs and/or spans with class name 'block-space-container'.
+  const blockSpaceContainers = xmlContainer.getElementsByClassName(
+    'block-space-container'
+  );
+  for (let i = blockSpaceContainers.length - 1; i >= 0; i--) {
+    blockSpaceContainers[i].remove();
+  }
```

Comment:
At first, I traced the call to the `convertXmlBlockly` function from the React component `MarkdownInstructions.jsx` (a component within the Instructions panel). And I just thought it would be more simple to remove the blockly container (div or span) instead of the SVGs. It's nice that we can now remove the other fixes in the .haml files.

Reply:
Just want to document that the alternative approach works as well.
```
  const blocklySvgs = xmlContainer.querySelectorAll(
    'svg[class*="readOnlyBlockSpace"]'
  );
   blocklySvgs.forEach(svg => {
     svg.parentNode.remove();
   });
```